PowerPointLabs/PowerPointLabs/ActionFramework/TooltipsLab/CreateTooltip/CreateTooltipEnabledHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿
+using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.ActionFramework.Common.Interface;
+using PowerPointLabs.Models;
+using PowerPointLabs.TextCollection;
+
+namespace PowerPointLabs.ActionFramework.TooltipsLab
+{
+    [ExportEnabledRibbonId(TooltipsLabText.CreateTooltipTag)]
+    class CreateTooltipEnabledHandler : EnabledHandler
+    {
+        protected override bool GetEnabled(string ribbonId)
+        {
+            PowerPointSlide currentSlide = this.GetCurrentSlide();
+            if (currentSlide != null)
```

Comment:
Is there a reason for not returning `currentSlide != nil` directly?

Reply:
thats true then less lines need, i'll make the changes!